layouts/shortcodes/home/call-to-actions.html

```diff
@@ -16,17 +15,17 @@
                 <div class="featured-section-cta-item-img">
                     <img src="/images/home/get-informed.svg" alt="Get Informed">
```

Comment:
We should update the alts as these can be read by screenreaders. Otherwise there may be English words on Chinese pages for screen readers which would be confusing

Reply:
Got it, it is a start and has a lot to go, including that the main menu for Chinese doesn't align with the English version, etc. We will fix that step by step.

Reply:
I'll hold off on merging until you give me the green light then?

Reply:
> I'll hold off on merging until you give me the green light then?

If that is OK, please merge these updates first, thanks.